hedyorg/hedy
# 🖊️ Add error for else statement without if statement #5099

Fixes #5099
Add a custom error in levels 5-7 when there is an `else` statement when there is no `if` right before it.

**How to test**
Run the following snippets in levels 5-7 and check that the custom error is appropriate:
```
if answer is yes print 'great!'
print 'correct'
else print 'wrong'
```
```
else print 'wrong'
```
